consensus/core/src/authority_service.rs

```diff
@@ -195,7 +195,7 @@ impl<C: CoreThreadDispatcher> NetworkService for AuthorityService<C> {
             })?;
         let verified_block = VerifiedBlock::new_verified(signed_block, serialized_block.block);
         let block_ref = verified_block.reference();
-        debug!("Received block {} via send block.", block_ref);
+        trace!("Received block {} via send block.", block_ref);
```

Comment:
This should be kept at debug too.

Reply:
Not sure it offers too much value at `debug` but I would always err on the side of keeping it if we believe so